docs/databases/swift-kuery.html

```diff
@@ -92,9 +99,10 @@ <h2 class="heading-2"><span class="blue-text">Step 2c:</span> Create connection
           <p>Add SwiftKuery and SwiftKueryPostgreSQL to your package.swift</p>
```

Comment:
Why is Step 2c for PostgreSQL called "Create connection pool of PostgreSQL connections" when the other two databases have a much simpler and easier to read title for the same section of "Create a connection to an SQLite database" can we have that here too please?